vroom/buildifier.vroom

```diff
@@ -52,3 +52,22 @@ The bzl filetype will use the buildifier formatter by default.
   $ )
 
   :set filetype=
+
+Errors are reported using the quickfix list.
+
+  @clear
+  % #<CR>
+  |java_package(])
```

Comment:
Nit: I think it's clearer as
```
  % #<CR>
  % java_package(])
```
like https://github.com/google/vroom/blob/master/examples/buffer.vroom does. I had to think more about the vroom syntax in the line continuation form.